### ISSUE TITLE
'form' dependency in gen.sh

### ISSUE THREAD

**Issue by N5FPP** [jmesmon/stm32f7xx#9]
What is the executable 'form' in the gen.sh? The only 'form' package I could find under Ubuntu 18.04 was for 'Symbolic manipulation system'.

**Comment by N5FPP** [comment]
Of course, as soon as I hit the return key I think to myself, check 'cargo search form' and there it is right at the top. You really need a Readme.md :)

**Comment by codyps** [comment]
That's an accurate statement :grimacing:
